src/dappsExplorer/DappCard.tsx

```diff
@@ -16,27 +16,26 @@ import fontStyles from 'src/styles/fonts'
 import { vibrateSuccess } from 'src/styles/hapticFeedback'
 import { Shadow, Spacing } from 'src/styles/styles'
 
-interface Props {
+interface DappCardContentProps {
   dapp: DappV1 | DappV2
+  onFavoriteDapp?: (dapp: DappV1 | DappV2) => void
+}
+
+interface Props extends DappCardContentProps {
   section: DappSection
   onPressDapp: (dapp: ActiveDapp) => void
-  onFavoriteDapp?: (dapp: DappV1 | DappV2) => void
 }
 
 // Since this icon exists within a touchable, make the hitslop bigger than usual
 const favoriteIconHitslop = { top: 20, right: 20, bottom: 20, left: 20 }
 
-function DappCard({ dapp, section, onPressDapp, onFavoriteDapp }: Props) {
+export function DappCardContent({ dapp, onFavoriteDapp }: DappCardContentProps) {
```

Comment:
What's the difference with `DappCard`?

Reply:
😅 DappCard has a border on all sides + border radius + an elevation shadow. DappCardContent is everything inside - the DappRankings card only has a bottom border but i wanted to reuse everything else